### PR TITLE
Challenge/david-vazquez-tour-of-heroes-react

### PR DESCRIPTION
**Pull request by davzlum**
… work

### CODE REVIEW

File: david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function Dashboard() {
+  const heroes = [
```
Comment (JDSanchon) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/dashboard/Dashboard.jsx:4]: Entiendo que esa constante deberia ser un componente que se imoporte allá donde sea necesario.

File: david-vazquez/challenges/react-tour-of-heroes/my-app/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import './App.css';
+import React from 'react';
+import Dashboard from './components/dashboard/Dashboard';
+import List from './components/list/List';
+import Details from './components/details/details';
+
```
Comment (JDSanchon) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/src/App.jsx:6]: Grande! Dale caña a esa lista! :)

File: david-vazquez/challenges/react-tour-of-heroes/my-app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
```
Comment (fredyzz) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/public/index.html:25]: Please delete the unnecessary comments

File: david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/details/details.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  const listItems = heroes.map((heroe) => (
+    <li>
+      El heroe
+      {' '}
+      {heroe.name}
+      {' '}
+      tiene la id:
+      {' '}
+      {heroe.id}
```
Comment (fredyzz) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/details/details.jsx:25]: You can use template literales for this purpose

File: david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/list/List.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    { id: 11, name: 'Dr Nice' },
+    { id: 12, name: 'Narco' },
+    { id: 13, name: 'Bombasto' },
+    { id: 14, name: 'Celeritas' },
+    { id: 15, name: 'Magneta' },
+    { id: 16, name: 'RubberMan' },
+    { id: 17, name: 'Dynama' },
+    { id: 18, name: 'Dr IQ' },
+    { id: 19, name: 'Magma' },
+    { id: 20, name: 'Tornado' },
```
Comment (fredyzz) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/list/List.jsx:14]: You should save this constant in another file, and import it when you need ir. DRY

**Review by fredyzz — APPROVED** [review]

File: david-vazquez/challenges/react-tour-of-heroes/my-app/.eslintrc.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
```
Comment (annasm07) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/.eslintrc.js:5]: jest: true

File: david-vazquez/challenges/react-tour-of-heroes/my-app/src/App.css [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
```
Comment (annasm07) [inline david-vazquez/challenges/react-tour-of-heroes/my-app/src/App.css:14]: unnecesary css be careful

**Review by annasm07 — APPROVED** [review]
Good Job!

**Review by shd8 — APPROVED** [review]
Good exercise David.

**Review by Emerson-feral — APPROVED** [review]

**Pull request merged**